volumina/imageView2D.py

```diff
@@ -137,14 +138,14 @@ def __init__(self, parent, cropModel, imagescene2d):
         self._crossHairCursor = CrossHairCursor(self.scene())
         self._crossHairCursor.setZValue(99)
 
-        posModel = self.scene()._posModel
-        axis = self.scene()._along[1] - 1 # axis is 0,1,2 for X,Y,Z
-        self._sliceIntersectionMarker = SliceIntersectionMarker(self.scene(), axis, posModel)
+        self.posModel = self.scene()._posModel
+        self.axis = self.scene()._along[1] - 1 # axis is 0,1,2 for X,Y,Z
+        self._sliceIntersectionMarker = SliceIntersectionMarker(self.scene(), self.axis, self.posModel)
```

Comment:
OK, I get it.  You're referencing them within `sliceSelectorHud.py`.  I guess that's fine.